lib/checks/forms/submit-button.js

```diff
@@ -0,0 +1,13 @@
+if (
+	node.querySelector(
```

Comment:
I don't think this works across shadow DOM boundaries. If you've got a form element in the light DOM and a submit button in a shadow DOM node inside that form, this won't return it. You can use `axe.utils.querySelectorAll` instead. That will do it.